### PR TITLE
Vg3

### PR DESCRIPTION
**Pull request by domoritz**
Migration to Vega 3.

### CODE REVIEW

File: vega-lite-schema.json [hunk 1]
```diff
@@ -2036,43 +2036,13 @@
...
-                            ],
-                            "type": "string"
-                        },
-                        {
-                            "type": "string"
-                        }
-                    ],
-                    "description": "The default symbol shape to use. One of: `\"circle\"` (default), `\"square\"`, `\"cross\"`, `\"diamond\"`, `\"triangle-up\"`, or `\"triangle-down\"`, or a custom SVG path."
+                    "description": "The default symbol shape to use. One of: `\"circle\"` (default), `\"square\"`, `\"cross\"`, `\"diamond\"`, `\"triangle-up\"`, or `\"triangle-down\"`, or a custom SVG path.",
+                    "type": "string"
```
Comment (domoritz) [inline vega-lite-schema.json]: @kanitw This is a actually a reasonable optimization but I wanted to call it out to you.

Comment (kanitw) [inline vega-lite-schema.json]: I don't know what you mean here.  Can you explain more?

Comment (domoritz) [inline vega-lite-schema.json]: Instead of the enum, we now just have the type `string`.

Comment (kanitw) [inline vega-lite-schema.json]: Ah make sense now.

**Pull request merged**